user_docs/en/changes.t2t

```diff
@@ -10,6 +10,8 @@ What's New in NVDA
 - Added support for Bluetooth Low Energy HID Braille displays. (#15470)
 - The Add-on Store now supports installing add-ons in bulk by selecting multiple add-ons. (#15350)
 - From the add-on store, it's possible to open a dedicated webpage to see or provide feedback about the selected add-on. (#15576, @nvdaes)
+- A new "on-demand" speech mode has been added.
+When speech is on-demand, NVDA does not speak automatically (e.g. when moving the cursor) but still speaks when calling commands whose goal is explicitly to report something (e.g. report window title). (#481, @CyrilleB79)

```

Comment:
Should this be referencing the issue number or the PR number?

Reply:
I have put the issue number as it is usually done. If you see a good reason to use the PR number here instead, please let it know.
In any case, someone who want to find the PR will find the reference in the issue.